google-cloud-storage/src/test/java/com/google/cloud/storage/ITGrpcMetricsTest.java

```diff
@@ -0,0 +1,84 @@
+package com.google.cloud.storage;
+
+import com.google.cloud.storage.it.runner.StorageITRunner;
+import com.google.cloud.storage.it.runner.annotations.Backend;
+import com.google.cloud.storage.it.runner.annotations.CrossRun;
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.sdk.metrics.SdkMeterProvider;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+@RunWith(StorageITRunner.class)
+@CrossRun(
+    backends = {Backend.PROD},
+    transports = {TransportCompatibility.Transport.GRPC})
+public class ITGrpcMetricsTest {
+  @Test
+  public void testGrpcMetrics() {
+    GlobalOpenTelemetry
+        .resetForTest(); // avoids problems with "GlobalOpenTelemetry.set has already been called"
+    Storage storage = StorageOptions.grpc().build().getService();
+    Assert.assertEquals(
+        "monitoring.googleapis.com:443",
+        ((GrpcStorageOptions) storage.getOptions())
+            .getCloudMonitoringEndpoint("storage.googleapis.com:443"));
+    SdkMeterProvider provider =
+        ((GrpcStorageOptions) storage.getOptions())
+            .createMeterProvider("monitoring.googleapis.com:443");
```

Comment:
I'm partial to using Truth for assertions rather than junit 4 assertions.

```suggestion
    GrpcStorageOptions grpcStorageOptions = StorageOptions.grpc().build();
    assertThat(grpcStorageOptions.getCloudMonitoringEndpoint("storage.googleapis.com:443"))
        .isEqualTo("monitoring.googleapis.com:443");

    SdkMeterProvider provider =
        grpcStorageOptions.createMeterProvider("monitoring.googleapis.com:443");

```